app/routers/apis.py

```diff
@@ -244,25 +244,36 @@ async def deregister_collators(
     return PlainTextResponse('OK')
 
 
-@router.post("/collators/{para_id}/add_invulnerables")
```

Comment:
I would prefer you keep the old Post endpoints (`add_invulnerables` and `remove_invulnerables`) but just mark them as `deprecated=True` (https://fastapi.tiangolo.com/tutorial/path-operation-configuration/#deprecate-a-path-operation). You can add the new endpoints under another path, eg. `/collators/{para_id}/invulnerables/add` and `/collators/{para_id}/invulnerables/remove`.